backend/src/db/migrations/20250915113733_feature-identity-groups.ts

```diff
@@ -0,0 +1,87 @@
+import { Knex } from "knex";
```

Comment:
IGNORE PLACEMENT OF COMMENT

Clicking into a project machine identity group leads to a broken page
<img width="1512" height="849" alt="image" src="https://github.com/user-attachments/assets/3162868b-1c52-4ffa-a6b5-938099f65eab" />
<img width="1512" height="853" alt="image" src="https://github.com/user-attachments/assets/26ef7e34-b6c7-4830-a8d0-f407955152c7" />